package.json

```diff
@@ -16,25 +16,33 @@
     "lib/"
   ],
   "scripts": {
-    "build": "browserify -d -r ./index.js:hydrolysis -o hydrolysis.js",
-    "build:watch": "watch 'npm run build' ./lib",
-    "release": "browserify -r ./index.js:hydrolysis -o hydrolysis.js",
+    "init": "typings install",
+    "clean": "rm -rf node_modules/ typings/ lib/ && find src | grep \\.js$ | xargs rm",
+    "build": "tsc && babel -q -d lib src && browserify -d -r ./index.js:hydrolysis -o hydrolysis.js",
```

Comment:
My goal here was to keep back-compat with the lib directory's layout as it was previously